docs/tutorial/progress-bar.md

```diff
@@ -1,35 +1,56 @@
 # Progress Bar in Taskbar (Windows, macOS, Unity)
 
-On Windows a taskbar button can be used to display a progress bar. This enables
-a window to provide progress information to the user without the user having to
-switch to the window itself.
+## Overview
```

Comment:
Can we add a simple paragraph about what this feature does and then segue into the platform-specific details?